lib/algo/DMD.h

```diff
@@ -74,11 +74,13 @@ class DMD
     /**
      * @brief Constructor. Basic DMD with uniform time step size.
      *
-     * @param[in] dim          The full-order state dimension.
-     * @param[in] dt           The dt between samples.
-     * @param[in] state_offset The state offset.
+     * @param[in] dim              The full-order state dimension.
+     * @param[in] dt               The dt between samples.
+     * @param[in] alt_output_basis Whether to use the alternative basis for output, i.e. phi = U^(+)*V*Omega^(-1)*X.
```

Comment:
This comment, repeated in many places, makes the line well over 100 characters. I think it would be better style to split the line after the word `for`, to keep 2 lines under 80 characters.

Reply:
@siuwuncheung Let's address this soon and merge this PR to master so that Dylan can use it for his DMD simulations.